classes/person.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_accessor :age, :name
+  attr_reader :id, :parent_permission
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = generate_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    # @nameable = nameable
```

Comment:
- [ ] It is unprofessional to leave commented code in your codebase so i kindly suggest you remove any unused code from your code base and keep it clean :+1: :sparkles: